### PR TITLE
Bump ethereumjs-wallet

### PR DESCRIPTION
**Pull request by FrederikBolding**
Bump `ethereumjs-wallet` to get rid of `scrypt` dependency. 

Needs more testing

### CODE REVIEW

**Comment by FrederikBolding** [comment]
[ch6798]

File: src/types/vendor/ethereumjs-wallet.d.ts [hunk 1]
```diff
@@ -1,26 +1,15 @@
 /**
```
Comment (vilsbole) [inline src/types/vendor/ethereumjs-wallet.d.ts:1]: Can remove?

File: src/types/vendor/ethereumjs-wallet.d.ts [hunk 2]
```diff
@@ -1,26 +1,15 @@
...
  * Incomplete -- Provider-engine needs its own typings
  */
-declare module 'ethereumjs-wallet/provider-engine' {
-  import { IPublicKeyOnlyWallet, IFullWallet } from 'ethereumjs-wallet';
-  class WalletSubprovider {
-    constructor(wallet: IPublicKeyOnlyWallet | IFullWallet, opts?: object);
-  }
-}
 
 declare module 'ethereumjs-wallet/thirdparty' {
```
Comment (vilsbole) [inline src/types/vendor/ethereumjs-wallet.d.ts:5]: Is the new version copied from `ethereumjs-wallet`?
Do you know if 0.6.5 includes the type changes, if so we can try to remove the declaration file.

**Review by vilsbole — CHANGES_REQUESTED** [review]

File: src/types/vendor/ethereumjs-wallet.d.ts (hunk 1 above)
Comment (FrederikBolding) [reply src/types/vendor/ethereumjs-wallet.d.ts:1]: We don't import `ethereumjs-wallet/provider-engine`, so I removed the type.

File: src/types/vendor/ethereumjs-wallet.d.ts (hunk 2 above)
Comment (FrederikBolding) [reply src/types/vendor/ethereumjs-wallet.d.ts:5]: It is not copied from `ethereumjs-wallet`, and `0.6.5` does not have type information. I've just modified what we had to fit the new version.

File: src/types/vendor/ethereumjs-wallet.d.ts (hunk 1 above)
Comment (vilsbole) [reply src/types/vendor/ethereumjs-wallet.d.ts:1]: I meant to ask if can remove the comment.
Apologies for the ambiguity.

**Review by vilsbole — APPROVED** [review]

**Pull request merged**